Backend/dev/migrations/rider-app/1116-rating.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE atlas_app.rating (
+id character(36) NOT NULL,
+ride_id character varying(36) NOT NULL,
+rating_value bigint,
```

Comment:
rating value is NULL so it should be of maybe type

Reply:
using int instead of bigint